sopel/config/core_section.py

```diff
@@ -1233,7 +1249,9 @@ def homedir(self):
 
     ssl_minimum_version = ValidatedAttribute(
         'ssl_minimum_version',
-        default="TLSv1_2",
+        default=TLSVersion.TLSv1_2,
+        parse=_parse_ssl_version,
+        serialize=lambda ver: ver.name,
```

Comment:
If you want, you can do `operator.attrgetter('name')` instead of the lambda. You'll need to add `import operator`.

Reply:
Hmm. Neat, but is that cleaner or more understandable than the lambda?

Reply:
Lambda has a weird place in Python. It's both loved and hated. Flake8, for instance, will tell you not to use them when you can use a `def` statement, but otherwise won't complain. Pylint may complain, but its rules are not clear to me yet.

I love `attrgetter` because it's a very functional approach. If you like it, use it. If you don't, leave the lambda. I'm only offering an option here. 😄